zk.js/src/merkleTree/merkleTreeConfig.ts

```diff
@@ -68,29 +68,20 @@ export class MerkleTreeConfig {
 
     const merkleTreeAuthorityAccountInfo =
       await this.getMerkleTreeAuthorityAccountInfo();
-    const transactionMerkleTreeIndex =
-      merkleTreeAuthorityAccountInfo.transactionMerkleTreeIndex;
-    const eventMerkleTreeIndex =
-      merkleTreeAuthorityAccountInfo.eventMerkleTreeIndex;
-
-    const oldTransactionMerkleTree =
-      MerkleTreeConfig.getTransactionMerkleTreePda(
-        transactionMerkleTreeIndex.sub(BN_1),
-      );
-    const newTransactionMerkleTree =
-      MerkleTreeConfig.getTransactionMerkleTreePda(transactionMerkleTreeIndex);
-    const oldEventMerkleTree = MerkleTreeConfig.getEventMerkleTreePda(
-      eventMerkleTreeIndex.sub(BN_1),
+    const merkleTreeSetIndex =
+      merkleTreeAuthorityAccountInfo.merkleTreeSetIndex;
+
+    const oldMerkleTreeSet = MerkleTreeConfig.getMerkleTreeSetPubkey(
+      merkleTreeSetIndex.sub(BN_1),
     );
-    const newEventMerkleTree =
-      MerkleTreeConfig.getEventMerkleTreePda(eventMerkleTreeIndex);
+    const newMerkleTreeSet =
+      MerkleTreeConfig.getMerkleTreeSetPubkey(merkleTreeSetIndex);
 
     const tx = await this.merkleTreeProgram.methods
-      .initializeNewMerkleTrees(DEFAULT_MERKLE_TREE_LOCK_DURATION)
```

Comment:
Just making sure we don't have lock duration anywhere else anymore either right?

Reply:
Nope, after this PR we shouldn't :sweat_smile: